simplified-app-shared/src/main/res/values/strings.xml

```diff
@@ -252,6 +252,9 @@
   <string name="audio_book_player">Audiobook Player</string>
   <string name="audio_book_player_error_engine_open">Unable to initialize audio engine</string>
   <string name="audio_book_player_error_book_open">Unable to open audio book</string>
+  <string name="audio_book_player_return">Return book?</string>
```

Comment:
Does this copy match the reference on the iOS implementation?

Reply:
Oops. Can you point me in the direction of the right strings?

Reply:
Title:
Your Audiobook Has Finished
Message:
Would you like to return it?

Reply:
Fixed in the latest commit.